code/game/objects/items/weapons/storage/toolbox.dm

```diff
@@ -82,7 +83,7 @@
 	new /obj/item/weapon/crowbar/red(src)
 	new /obj/item/weapon/wirecutters(src, "red")
 	new /obj/item/device/multitool(src)
-	new /obj/item/clothing/gloves/color/red/insulated(src)

```

Comment:
Don't, though, I've got a pull to change the red gloves sprite to be a bit more, ah, red and less pale and shitty.